tests/unit/utils/test_version.py

```diff
@@ -950,3 +953,74 @@ def test_opengl_vendor():
     """Simply call version.opengl_vendor() and see if it doesn't crash."""
     pytest.importorskip("PyQt5.QtOpenGL")
     return version.opengl_vendor()
+
+
+class HTTPPostStub(QObject):
```

Comment:
Instead of duplicating the code from `test_pastebin.py`, can you please move it to some common place like `helpers/stubs.py`? You can then access that module using the `stubs` fixture.